lib/src/clojure_lsp/feature/hover.clj

```diff
@@ -65,6 +65,37 @@
                              (map #(str %))
                              (string/join "\n---\n"))))))
 
+(defn recurse-doc-line
+  [db markdown? uri doc cnt]
+  (cond
+    (string? doc)
+    (when (seq doc)
+      (if markdown?
+        (docstring->formatted-markdown doc)
+        doc))
+    (< 2 cnt) nil
+    ;; special case for `(:doc (meta #'some-var))`
+    (seq? doc)
+    (let [referenced-var-meta
+          (and (seq? doc)
+               (= :doc (first doc))
+               (let [doc' (fnext doc)]
+                 (and (seq? doc')
+                      (= 'meta (first doc'))
+                      (let [doc'' (fnext doc')]
+                        (and (= 'var (first doc''))
+                             (meta (second doc'')))))))
+          referenced-var-docs
+          (when referenced-var-meta
+            (->> (q/find-element-under-cursor
+                   db uri
+                   (:row referenced-var-meta)
+                   (:col referenced-var-meta))
+                 (q/find-definition db)
```

Comment:
I think you can use directly `q/find-definition-from-cursor`

Reply:
Thanks, I didn't see that one.